easybuild/easyconfigs/m/MIGRATE-N/migrate-3.6.11_icc.patch

```diff
@@ -0,0 +1,13 @@
+#icc does not find -lpthread, if -fast defined
+#B. Hajgato 3rd September 2015
+--- src/configure.old	2015-05-02 14:42:10.000000000 +0200
++++ src/configure	2015-09-03 10:00:54.320014335 +0200
+@@ -3333,7 +3333,7 @@
+      	     pgcc) CFLAGS="-B -fast ";
+ 	     	  SYSTEM="gcc";;
+ #            icc)  CFLAGS=" -O3 -tpp7 -xKW -pad -ip ";
```

Comment:
It automatically picks the $CFLAGS env. var, It could be actually deleted that line (or commented out)